File: app/src/main/java/com/xfhy/rinlinedemo/MainActivity.java
package com.xfhy.rinlinedemo;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;

import com.xfhy.mylibrary.MyActivity;
import com.xfhy.mylibrary2.MySecondActivity;

/**
 * @author : xfhy
 * Create time : 2023/9/17 20:29
 * Description :
 */
public class MainActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        startActivity(new Intent(this, MyActivity.class));
        startActivity(new Intent(this, MySecondActivity.class));
    }
}

File: app/src/main/java/com/xfhy/rinlinedemo/MainActivity.kt
package com.xfhy.rinlinedemo

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.xfhy.mylibrary.MyActivity
import com.xfhy.mylibrary2.MySecondActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        startActivity(Intent(this, MyActivity::class.java))
        startActivity(Intent(this, MySecondActivity::class.java))
    }
}